### DISCUSSION TITLE
Upgrading from 19.4.x to 20: role resources are no longer available, expected behaviour?

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3904] [Questions & Answers] [answered]
When I log in to the admin, I am getting this notification.

The following role resources are no longer available in the system: admin/system/api/consumer, admin/system/api/consumer/edit, admin/system/api/consumer/delete, admin/system/api/authorizedTokens, admin/cms/poll. You can delete them by [clicking here]

**Comment by kiatng** [comment, marked as answer]
Everything is explained in PR #3647
